src/components/question-editor/QuestionEditor.tsx

```diff
@@ -130,63 +156,108 @@ const QuestionEditor = () => {
     </Box>
   ))
 
+  const disabledButton =
+    type === 'openAnswer' ? text && openAnswer : text && answers[0]?.text
+
+  const buttons = (
+    <Box sx={styles.buttons}>
+      <AppButton onClick={() => navigate(-1)} variant={ButtonVariantEnum.Tonal}>
+        {t('common.cancel')}
+      </AppButton>
+      <AppButton
+        disabled={!disabledButton}
```

Comment:
it should be disabled={disabledButton}, so refator disabledButton statement

Reply:
Changed